runtime/src/launch/amudprun/launch-amudprun.c

```diff
@@ -97,13 +97,34 @@ static char** chpl_launch_create_argv(const char *launch_cmd,
     const char* ev_use_lldb = getenv("CHPL_COMM_USE_LLDB");
 
     if (ev_use_gdb != NULL || ev_use_lldb != NULL) {
+
+      // determine the terminal emulator to use
+      char* dbg_term = getenv("CHPL_COMM_DBG_TERM");
```

Comment:
When I built this branch with `WARNINGS=1` I got this from gcc, on the assignments at lines 104 and 116:
> launch-amudprun.c:104:18: error: assignment discards ‘const’ qualifier from pointer target type [-Werror=discarded-qualifiers]

Changing this declaration to `const char* dbg_term ...` got rid of those errors.  Would you make this change, please?

Reply:
Done.